### ISSUE TITLE
Project seetings are not accessable

### ISSUE THREAD

**Issue by AlUser71** [akiko-pusu/redmine_banner#127]
By using the redmine_bunner plugin in version 0.2.2 or 0.3.0 the project settings can't be accessed anymore.

It seems that it is caused by issue #120 by changing lib/banners/projects_helper_patch.rb.
![projects_helper_patch rb](https://user-images.githubusercontent.com/60700838/73848971-685ce980-4829-11ea-956a-79f106a9c1b2.PNG)

If we patch `projects_helper_patch.rb` and replace again
`ProjectsHelper.prepend(Banners::ProjectsHelperPatch)`
by 
`ProjectsController.helper(Banners::ProjectsHelperPatch)`
it works again.

Our patched version looks like this now:
```
unless ProjectsHelper.included_modules.include?(Banners::ProjectsHelperPatch)
  ProjectsController.helper(Banners::ProjectsHelperPatch)
end
```
I don't know if there are side effects that I could not figure out so far.

**Comment by akiko-pusu** [comment]
Hi, @AlUser71 
Thanks for your reporting.

> I don't know if there are side effects that I could not figure out so far.

It depends on your environment.
Could you please more detailed information about your environment?
Such as Redmine version, installed plugins and so on.

Some plugins often conflict with each other.

**Comment by AlUser71** [comment]
Hi, @akiko-pusu 
please find attched our Redmine configuration.

For us it is important to know, if the change/patch that we applied to your code (projects_helper) is OK from your point of view (https://github.com/akiko-pusu/redmine_banner/issues/127#issue-560398218).

![grafik](https://user-images.githubusercontent.com/60700838/74144737-906b9480-4bfd-11ea-9bd8-b7281277b5b4.png)

**Comment by akiko-pusu** [comment]
Thanks for your additional information.
Let me confirm,  is your Redmine instance clean installed one or not?
 
It's hard for me to find the plugin that conflicts with Redmine Banner.
easy_xxxx series 

And I'm going to refactor not to use project setting tab like this:

- https://github.com/akiko-pusu/redmine_issue_templates/pull/299

BTW, thanks for using Issue Template plugin v1.0.0.

**Comment by AlUser71** [comment]
I understand that you can't check or validate your plugin against any other plugin, that's OK.

But what we would like to know is:
Is the change that we have applied OK from your point of view regarding the banner plugin or not? Currently we don't have any problem by using our changed version.

- Code in **projects_helper_patch.rb** in version 0.3.0, not working in our environment (project setting pages not accesable):

```
unless ProjectsHelper.included_modules.include?(Banners::ProjectsHelperPatch)
  ProjectsHelper.prepend(Banners::ProjectsHelperPatch) )
end
```

- Our modified version, works in our environment:

```
unless ProjectsHelper.included_modules.include?(Banners::ProjectsHelperPatch)
  ProjectsController.helper(Banners::ProjectsHelperPatch)
end
```

**Comment by AlUser71** [comment]
Sorry, closing the issue was a mistake.

**Comment by akiko-pusu** [comment]
Hi, I've modified not to use the project setting tab but to use the project menu to access the banner edit screen. (Please see: #128)

Now, this update is included in the dev branch and soon it will be merged into the master.

**Comment by akiko-pusu** [comment]
Released v0.3.1. 
(No migration is required, so please try if you have time!)

**Comment by AlUser71** [comment]
> Released v0.3.1.
> (No migration is required, so please try if you have time!)

We have testet version 0.3.1. It is working now and we have seen no further conflict.
Thanks for you quick support.
